coral/src/app/main.module.css

```diff
@@ -22,6 +22,14 @@ body {
   -webkit-text-size-adjust: 100%;
 }
 
+/*!* removes the "x" in search inputs from webkit *!*/
```

Comment:
Was this an experiment that didn't pan out? SHould it be uncommented or deleted? ^^

Reply:
Oh, good catch! That actually did pan out, I really have no idea why I commented it out though  🤦 

This is how it looked like before:
<img width="479" alt="Screenshot 2023-11-20 at 10 32 10" src="https://github.com/Aiven-Open/klaw/assets/943800/03db38f7-9bf8-4afd-8898-ec0ac62aabcb">


And this is how it looks now:

<img width="505" alt="Screenshot 2023-11-20 at 10 33 06" src="https://github.com/Aiven-Open/klaw/assets/943800/d6c96514-88ba-4e50-982c-362738947baf">


It only was an issue in webkit based browsers, Firefox looked good. It may have been a more recent change in browsers, as that's the first time I've seen this 🤔

Reply:
https://github.com/Aiven-Open/klaw/pull/1971/commits/1265d000ff11eccd14340530b1645c10a831e875

Reply:
Wonder if it's a bug in `aquarium` that has been fixed in a later version? We're like 4  minor versions behind haha! Because we should be seeing the same in Storybook, but I don't see it 🤔

Reply:
Aquarium uses a input type `text` for their search, the style comes from the type `search` that we're using.